pyproject.toml

```diff
@@ -116,7 +116,10 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pyright]
-include = ["src/backend/services/metrics.py"]
+include = [
+  "src/backend/services/metrics.py", 
+  "src/backend/tools/google_drive/sync/actions/",
```

Comment:
What does this help with, just curious?

Reply:
i believe tanzim introduced this pyright static type checking. so everything included here would be checked